webapp/pages/settings/my-email-address/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <ds-card centered v-if="success">
+    <transition name="ds-transition-fade">
+      <sweetalert-icon icon="info" />
+    </transition>
+    <ds-text v-html="submitMessage" />
+  </ds-card>
+  <ds-form v-else v-model="form" :schema="formSchema" @submit="submit">
+    <template slot-scope="{ errors }">
+      <ds-card :header="$t('settings.email.name')">
+        <ds-input
+          id="email"
+          model="email"
+          icon="envelope"
+          :label="$t('settings.email.labelEmail')"
+        />
+
+        <template slot="footer">
+          <ds-space class="backendErrors" v-if="backendErrors">
+            <ds-text align="center" bold color="danger">{{ backendErrors.message }}</ds-text>
+          </ds-space>
+          <ds-button icon="check" :disabled="errors" type="submit" primary>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-02T09:28:37Z" title="Wednesday, October 2nd 2019, 11:28:37 am +02:00">Oct 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,113 @@
+<template>
+  <ds-card centered v-if="success">
+    <transition name="ds-transition-fade">
+      <sweetalert-icon icon="info" />
+    </transition>
+    <ds-text v-html="submitMessage" />
+  </ds-card>
+  <ds-form v-else v-model="form" :schema="formSchema" @submit="submit">
+    <template slot-scope="{ errors }">
+      <ds-card :header="$t('settings.email.name')">
+        <ds-input
+          id="email"
+          model="email"
+          icon="envelope"
+          :label="$t('settings.email.labelEmail')"
+        />
+
+        <template slot="footer">
+          <ds-space class="backendErrors" v-if="backendErrors">
+            <ds-text align="center" bold color="danger">{{ backendErrors.message }}</ds-text>
+          </ds-space>
+          <ds-button class="submit-button" icon="check" :disabled="errors" type="submit" primary>
```


the `class` is now obsolete, no?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-02T13:38:56Z" title="Wednesday, October 2nd 2019, 3:38:56 pm +02:00">Oct 2, 2019</time>_

---


Ok, you got me. I hoped to find not enough suggestions to touch this PR again, but this suggestion together with @mattwr18's suggestions above is too much to just click "Merge"...